iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_queue_popper.inl

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
I guess no changes here, only a rename from cpp to inl...

Reply:
Corrrect. For some reason it's not identified as such for this file.